spec/factories/course_lesson_plan_items.rb

```diff
@@ -9,6 +9,8 @@
     end_at { nil }
     sequence(:title) { |n| "Example Lesson Plan Item #{n}" }
     published { false }
+    has_personal_times { true }
+    affects_personal_times { true }
```

Comment:
The default value is false because of functionality for users -- if they didn't enable the feature themselves then we should not create personal times for it.

The default value for the factory is true for testing -- a lot of logic doesn't run if one of the booleans is false, but we want the logic to run during tests.